imessage/bluebubbles/interface.go

```diff
@@ -171,8 +171,15 @@ type SendTextResponse struct {
 }
 
 type ReadReceiptResponse struct {
-	Status  int64  `json:"status"`
-	Message string `json:"message"`
+	Status  int64       `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data"`
```

Comment:
I don't recall `data` being in this struct, did you add it on purpose?

Reply:
I actually meant to add one called `error` because when BlueBubbles returns an error, they put the specific information on the failure in that object. I got distracted when I added this field and added a `data` object instead.

For example:
```json
{"status":400,"message":"You've made a bad request! Please check your request params & body","error":{"type":"Validation Error","message":"A 'tempGuid' is required when sending via AppleScript"}}
```

So this is me trying to get ahead of errors to make sure they are logged just in case. Let me correct that name, then I'll merge this.